src/retro_data_structures/formats/ntwk.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+import typing
+
+import construct
+
+from retro_data_structures.base_resource import AssetType, BaseResource, Dependency
+from retro_data_structures.common_types import FourCC
+from retro_data_structures.formats.script_object import ConstructScriptInstance, ScriptInstance
+
+if typing.TYPE_CHECKING:
+    from retro_data_structures.game_check import Game
+
+NTWK = construct.Aligned(
+    32,
+    construct.Struct(
+        "magic" / construct.Const("NTWK", FourCC),
+        "version" / construct.Const(1, construct.Int8ub),
+        "script_instances" / construct.PrefixedArray(construct.Int32ub, ConstructScriptInstance),
+    ),
+)
+
+
+class Ntwk(BaseResource):
+    @classmethod
+    def construct_class(cls, target_game: Game) -> construct.Construct:
+        return NTWK
+
+    @classmethod
+    def resource_type(cls) -> AssetType:
+        return "NTWK"
+
+    def dependencies_for(self) -> typing.Iterator[Dependency]:
+        yield from []
```

Comment:
I don't think this is strictly true but it also doesn't affect paks or mlvls so whatever lol

Reply:
Could add a comment or even raise an Exception?